content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
+
+Group payment
+-------------
+
+Group payments allow to combine outgoing and incoming payments from the same partner into a single
+payment to simplify the process and improve payment usability. To register a group payment:
+
+#. Select the related invoices/credit notes or bills/refunds from one partner.
+#. In the :icon:`fa-cog` (:guilabel:`Action`) menu, select :guilabel:`Register Payment`.
+#. In the :guilabel:`Register Payment` window, select the :guilabel:`Journal`, the
+   :guilabel:`Payment Method`, and the :guilabel:`Payment Date`.
+#. Enable the :guilabel:`Group Payments` option.
+
+The invoices or bills are now marked as :guilabel:`In payment` until they are reconciled with the
+corresponding bank statements.
+The credit notes or refunds are marked as :guilabel:`Paid`.
+
+.. _accounting/payments/batch-payments:
 
 Batch payment
 -------------
 
-Batch payments allow you to group different payments to ease :doc:`reconciliation
-<bank/reconciliation>`. They are also useful when you deposit :doc:`checks <payments/checks>` to the
-bank or for :doc:`SEPA payments <payments/pay_sepa>`. To do so, go to :menuselection:`Accounting -->
-Customers --> Batch Payments` or :menuselection:`Accounting --> Vendors --> Batch Payments`. In the
-list view of payments, you can select and group several payments in a batch by clicking
-:menuselection:`Action --> Create Batch Payment`.
+Batch payments allow to group payments from multiple partners to ease
+:doc:`reconciliation <bank/reconciliation>`. They are also useful when depositing :doc:`checks
+<payments/checks>` to the bank or for :doc:`SEPA payments <payments/pay_sepa>`. To do so, go to
+:menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
+Vendors --> Batch Payments`. In the list view of payments, to select and group several payments in a
+batch, click :menuselection:`Action --> Create Batch Payment`.
 
 .. seealso::
   - :doc:`payments/batch`
   - :doc:`payments/batch_sdd`
 
-.. _payments/matching:
+.. _accounting/payments/matching:
 
 Payments matching
 -----------------
 
 The :guilabel:`Payments matching` tool opens all unreconciled customer invoices or vendor bills and
-allows you to process them individually, matching all payments and invoices in one place. You can
-reach this tool from the :menuselection:`Accounting Dashboard --> Customer Invoices / Vendor Bills`,
-click the drop-down menu button (:guilabel:`⋮`), and select :guilabel:`Payments Matching`, or go to
-:menuselection:`Accounting --> Accounting --> Reconciliation`.
+allows to process them individually, matching all payments and invoices in one place. Go to the
+:guilabel:`Accounting Dashboard`, and click the :icon:`fa-ellipsis-v` drop-down menu button from the
+:guilabel:`Customer Invoices` or :guilabel:`Vendor Bills` journals, and select
+:guilabel:`Payments Matching`, or go to :menuselection:`Accounting --> Accounting -->
+Reconciliation`.
 
 .. image:: payments/payments-journal.png
```

Comment:
It would be nice to update the screenshot either here or in 17

Reply:
@dade-odoo Sure, I will in 18